src/submission/models.py

```diff
@@ -220,6 +220,13 @@ def article_media_upload(instance, filename):
     STAGE_REJECTED,
 }
 
+SECTION_EDITOR_STAGES = {
+    STAGE_UNASSIGNED,
+    STAGE_ASSIGNED,
+    STAGE_UNDER_REVIEW,
+    STAGE_UNDER_REVISION
+}
+
```

Comment:
This is not really the case, section editors can handle other stages like copyediting. I think it is best to delete this, since it not needed anyways (as per my comment above)